File: openapi/code_samples/csharp/data/getonebank.cs
var client = new RestClient("{{base_url}}/api/v1/system-data/banks/0/");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Authorization", "{{auth_token}}");
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

File: openapi/code_samples/PHP/data/getonebank.php
<?php
$client = new Client();
$headers = [
  'Authorization' => '{{auth_token}}'
];
$request = new Request('GET', '{{base_url}}/api/v1/system-data/banks/0/', $headers);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();
